Configuration/TCA/Module/Options.php

```diff
@@ -85,6 +85,12 @@
                 'type' => 'check',
             ],
         ],
+        'auto_orient_custom' => [
+            'label' => 'Use custom auto_orient (flop rotate 6 and 8 handling, based on image exif)',
```

Comment:
I see the command to use with —flop but “flop rotate”, really? that doesn’t mean anything to normal people. It should be maybe “mirror and rotate” but then I wonder why we need some custom configuration option for that since exit info is supposed to describe what to be done. Do I miss some info of the actual problem?